### ISSUE TITLE
Add a GitHub Actions workflow to generate `DIRECTORY.md`

### ISSUE THREAD

**Issue by Panquesito7** [TheAlgorithms/Solidity#53]
**Description**

To make the algorithm addition process easier, we can make a GitHub Actions workflow to generate a `DIRECTORY.md` file.
Many repositories in the organization do this to keep things organized. We can use the workflow from the [**C++ repository**](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/.github/workflows/awesome_workflow.yml#L49).

What do you think? Thanks. 🙂

**Comment by mkubdev** [comment]
Yes, I agree to follow the [C++ repository](https://github.com/TheAlgorithms/C-Plus-Plus/) organization.
What changes need to be done?

**Comment by Panquesito7** [comment]
I'll be making a PR and you will be able to see the changes as well as how it works. I hope that's fine. 🙂

**Comment by mkubdev** [comment]
Yes, thanks a lot :)

**Comment by Panquesito7** [comment]
Will work on this once all the files/DSA matters are solved.
